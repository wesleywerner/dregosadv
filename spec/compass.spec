go faster
exit
north
southwest
again
again
again
examine compass
southwest
examine compass
