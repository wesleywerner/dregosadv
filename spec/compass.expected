   The Maro River                                 Time:  3:00 pm
  
  April 10th, 1940.
  
  Your journey started a few days ago after docking at Port Moresby, from
  there it was two day's ride on the back of a truck to Kerema, followed by a
  small aircraft flight and landing near the mouth of the Maro River.
  
  You met your guide and his canoe, who will take you up-river to your final
  destination -- a remote and relatively uncharted area in New Guinea -- to
  find the legendary Golden Egg of Man-Toomba
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 200410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
  You are in a canoe on the Maro river. Your guide is with you, rowing the
  craft.
. 
> >   River Shore                                    Time:  3:01 pm
. 
  You hasten your journey.
  
  "Here we are", your guide says -- the canoe comes to a stop on the shore.
  
  River Shore (in the canoe)
  The canoe has stopped ashore, you can get out now.
. 
> >   River Shore                                    Time:  3:02 pm
. 
  You get out of the canoe.
  
  River Shore
  You are on the river shore. A roughly cut path leads North into the jungle.
  
  Your guide sits in the canoe gazing out over the moving river, the salty
  water gently lapping the sides of his craft.
  
> >   Jungle Path                                    Time:  3:03 pm
. 
  Jungle Path
  Trees of all sizes and leaves of all shapes fill every available space of
  the jungle, this roughly cut path will be indistinguishable from the rest
  in a few months.
  
  The river shore is back South, you notice a clearing in the jungle to the
  West.
  
> >   Lost in the Jungle                             Time:  3:04 pm
. 
  Lost in the Jungle
  You don't know where you are.
  
> >   Lost in the Jungle                             Time:  3:05 pm
. 
  You fumble through the jungle.
  
> >   Lost in the Jungle                             Time:  3:06 pm
. 
  You fumble through the jungle.
  
> >   Lost in the Jungle                             Time:  3:07 pm
. 
  You fumble through the jungle.
  
> >   Jungle Path                                    Time:  3:08 pm
. 
  Your compass guides you back to safety.
  
  Jungle Path
  Trees of all sizes and leaves of all shapes fill every available space of
  the jungle, this roughly cut path will be indistinguishable from the rest
  in a few months.
  
  The river shore is back South, you notice a clearing in the jungle to the
  West.
  
  [The score has just gone up by one point.]
  
> >   Lost in the Jungle                             Time:  3:09 pm
. 
  Lost in the Jungle
  You don't know where you are.
  
> >   Jungle Path                                    Time:  3:10 pm
. 
  Your compass guides you back to safety.
  
  Jungle Path
  Trees of all sizes and leaves of all shapes fill every available space of
  the jungle, this roughly cut path will be indistinguishable from the rest
  in a few months.
  
  The river shore is back South, you notice a clearing in the jungle to the
  West.
  
> >